wellcomeml/ml/bert_semantic_equivalence.py

```diff
@@ -0,0 +1,176 @@
+import os
+
+import tensorflow as tf
+
+from transformers import (
+    BertConfig,
+    BertTokenizer,
+    TFBertForSequenceClassification,
+    glue_convert_examples_to_features,
+    glue_processors,
+)
+
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.model_selection import train_test_split
+
+from wellcomeml.ml.bert_vectorizer import check_cache_and_download
+
+
+class SemanticEquivalenceClassifier(BaseEstimator, TransformerMixin):
+    """
+    Class to fine-tune BERT-type models for semantic equivalence, for example
+    paraphrase, textual similarity and other NLU tasks
+    """
+    def __init__(self, pretrained='bert',
+                 batch_size=32,
+                 eval_batch_size=32*2,
```

Comment:
why different?

Reply:
Good question - I don't think these too are tied, and for small sets `eval_batch_size` can even be set to the entire dataset. I suppose `eval_batch_size` can be bigger because it doesn't to backpropagation (just a forward pass). For different datasets it will be a question of experimentation I guess.

Reply:
Interesting, i have not seen the differentiation before. I am not so sure this parameter changes anything, maybe it makes inference faster in case data fit into memory.